data-prepper-core/src/test/java/org/opensearch/dataprepper/peerforwarder/PeerForwarder_ClientServerIT.java

```diff
@@ -325,16 +340,32 @@ void send_Events_to_server_with_unknown_certificate_key_closes() {
             assertThat(receivedRecords, notNullValue());
             assertThat(receivedRecords, is(empty()));
         }
+
+        @Test
+        void send_Events_to_an_unknown_server_should_throw() {
+            final PeerForwarderConfiguration peerForwarderConfiguration = createConfiguration(
+                    true, ForwardingAuthentication.MUTUAL_TLS, ALTERNATE_SSL_CERTIFICATE_FILE, SSL_KEY_FILE, true);
```

Comment:
nit: Is the key supposed to be ALTERNATE_SSL_KEY_FILE here?

Reply:
No, but I see how this is confusing. I am intentionally using the trusted key. But, I'm configuring the client to trust the alternate key.